src/template-tags/tickets.php

```diff
@@ -123,6 +123,11 @@ function tribe_tickets_display_count_and_ticket_button() {
 		// get an array for ticket and rsvp counts
 		$types = Tribe__Tickets__Tickets::get_ticket_counts( $event_id );
 
+		// if no rsvp or tickets return
+		if ( ! $types ) {
+			return null;
```

Comment:
I do not get what you are saying here. 

The return for tribe_tickets_display_count_and_ticket_button is either a string or null